src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeProcessTests.cs

```diff
@@ -354,5 +354,14 @@ private static void SafeKillProcess(Process process)
                 // do nothing
             }
         }
+
+        private static string GetDefaultParameters()
+        {
+            var workDir = Path.Combine(Path.GetTempPath(), "SLVS", "ESLintBridge", "workdir");
```

Comment:
This working directory will be the same every instance of Visual Studio. Two related questions:

1) what happens if there are multiple instances of VS running?

2) what was the previous behaviour? i.e. if we passed nothing did the server generate a per-instance directory, or did it always use the same one?

Reply:
According to my talk with Tibor; 
1) It should be unique per project, so if two instances are open with different projects, it could create a problem; not sure if it's two instances of the same project. But I did not question it further because I've decided to go with a per-instance approach. 
2) This directory is used for only "security rules" at the moment, which are not available in SonarLint. So what we passed did not matter and does not matter now. But I've decided to implement a per-instance-based approach so if anything changes, it won't break.